### ISSUE TITLE
.AddIdentityWithMongoStoresUsingCustomTypes doesn't work corrrectly.

### ISSUE THREAD

**Issue by ayberkcanturk** [g0t4/aspnet-identity-mongo-sample#4]
After following service registeration;

services.AddIdentityWithMongoStoresUsingCustomTypes<ApplicationUser, IdentityRole>(_configuration.GetConnectionString("DefaultConnection"))
                .AddDefaultTokenProviders();

I've got following error;

InvalidOperationException: No service for type 'Microsoft.AspNetCore.Identity.SignInManager`1[cvdb.Service.IdentityServer.Models.ApplicationUser]' has been registered.

It looks like "AddIdentityWithMongoStoresUsingCustomTypes" doesn't provide the correct classes.

**Comment by Galilyou** [comment]
Although this is closed, I'm still getting it.  Has this been fixed? I'm doing the same thing as mentioned by @ayberkcanturk , am I doing something wrong here?

@g0t4

**Comment by g0t4** [comment]
I don't know what problem @ayberkcanturk had. I might be able to help if I can see your Startup.cs

**Comment by ayberkcanturk** [comment]
Hey @Galilyou, @g0t4 , 
There was a version issue with package versions. Could you please write them so we can check?

**Comment by Galilyou** [comment]
That, I think is the relevant part in my Startup.cs

```
 services.AddIdentityWithMongoStoresUsingCustomTypes<ApplicationUser, ApplicationRole>(Configuration.GetConnectionString("DefaultConnection"))
                .AddDefaultTokenProviders();
```

I'm simply trying to use an extended User and Role types. `ApplicationUser` extends `IdentityUser`  and `ApplicationRole` extends `IdentityRole`.

The package version was `"1.0.0-beta1"` and I changed it to `1.0.2`

I had to `dotnet migrate` the sample project, of course, but I don't assume this to have broken anything.

I will retry again in a fresh project .. 

@ayberkcanturk  @g0t4
